docs/scos/user/technology-partners/202204.0/payment-partners/unzer/install-and-configuration-unzer.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Installing and configuring Unzer.
+description: Install and configure Unzer module to work in the Spryker Commerce OS.
+last_updated: Jun 07, 2022
+template: concept-topic-template
+originalLink: 
+originalArticleId:
+related:
+- title: Unzer integration guide
+  link: docs/scos/dev/technology-partner-guides/page.version/payment-partners/unzer/project-integration-unzer.html
+---
+
+This topic describes how to install and configure the Unzer module to integrate Unzer into your project.
+
+## Integration overview
+
+The `spryker-eco/unzer` module provides integration of Spryker e-commerce system with Unzer technology partner. It requires the `SprykerEco.UnzerApi` `spryker-eco/unzer-api` module that provides the REST Client for making API calls to the Unzer Payment Provider.
+It also requires `SprykerEco.UnzerGui` `spryker-eco/unzer-gui` module that provides Backoffice functionality to configure Unzer credentials.
+
+The `SprykerEco.Unzer` module includes integration with:
+
+* Checkout process - payment forms with all necessary fields that are required to make payment request, save order information and so on.
+* OMS (Order Management System) - state machines, all necessary commands for making modification requests and conditions for changing orders status accordingly.
+
+The `SprykerEco.Unzer` module provides the following payment methods:
+* Credit Card
+* Sofort
+* Bank Transfer
+* Marketplace Credit Card
+* Marketplace Sofort
+* Marketplace Bank Transfer
+
+## Installation
+
+To install the Unzer modules, run the command:
+```
+composer require spryker-eco/unzer spryker-eco/unzer-gui
+```
+
+## General Configuration
+
```

Comment:
Would recommend to split this part into two subpoints

### Unzer configuration
### Unzer API configuration

As well shouldn't we list all possible configuration keys? Seems those from API are missing.